aries-site/src/pages/index.js

```diff
@@ -3,6 +3,7 @@ import { initialize, pageview } from 'react-ga';
 import { Config } from '../../config';
 
 import About from './start/about';
+// import Landing from './landing/landing';
```

Comment:
Is there a reason we don't want to just place the code for the index page in index.js? By creating a file in ./landing/landing, we are creating a route of aries.hpe.design/landing/landing which I don't think is a route we want people to visit. If we do want to keep the landing page code in a separate file for clarity in the file name, let's make sure to add the canonical link in the SEO that @halocline had mentioned.

Reply:
I agree. I'll make the change.

Reply:
Done.